### ISSUE TITLE
When splitting COCO dataset, what is considered as a sample?

### ISSUE THREAD

**Issue by hana9090** [dbolya/yolact#337]
I have a dataset with coco formate, and I want to split it into train and validation to train yolact model so should I have to consider each annotation as sample? Is that correct or not?

Thanks

**Comment by dbolya** [comment]
Yup, each annotation is a sample. So you can include the same images in both the validation set and training set, since only the images with annotations will be used.

**Comment by hana9090** [comment]
Thanks @dbolya for the response :) when I am done with splitting based on the number of annotations. In train.py I print `len(dataset) `and `len(val_dataset) ` So the total number of these on train are more than the original number of annotations.  
So the following line will make augmentation on sample as well? or it has to return only the exact coco dataset?
 
```
dataset = COCODetection(image_path=cfg.dataset.train_images,
                            info_file=cfg.dataset.train_info,
                            transform=SSDAugmentation(MEANS))
```

**Comment by dbolya** [comment]
Those lengths should be the number of images with at least one annotation, so that looks correct.

And in order to control augmentations, you don't need to change the code. It's all done in the config settings here:
https://github.com/dbolya/yolact/blob/f54b0a5b17a7c547e92c4d7026be6542f43862e7/data/config.py#L487-L499

You can turn on the last two if you want more augmentation.
